cocos/scripting/js-bindings/manual/network/jsb_websocket.cpp

```diff
@@ -78,6 +79,9 @@ class JSB_WebSocketDelegate : public WebSocket::Delegate
         js_proxy_t * p = jsb_get_native_proxy(ws);
         if (!p) return;
 
+        if (cocos2d::Director::getInstance() == nullptr || cocos2d::ScriptEngineManager::getInstance() == nullptr)
```

Comment:
Could Director::getInstance be nullptr? It will always return non-null pointer in cocos2d-x.
I think it's a copy paste logic from cocos2d-x-lite since I know that lite does it in this way.